### PR TITLE
add shared ip faqs

### LINKED ISSUE #1328: Add Shared Ip to Networks FAQ
To be merged on August 15th:

The content from the ether pad needs to be added to the shared networks FAQ.

The content for the Shared IP LA FAQs is here:
https://etherpad.rax.io/p/sharedipfaq

To be added to the BOTTOM OF the Cloud Networks FAQs page here:
https://support.rackspace.com/how-to/cloud-networks-faq/

Please let me know if you have any questions, or require anything from me to help in tracking this, etc.

Also, I will look through the contributing guide sent by Cat as well, just to help me help you in the future :-).

Thank you both for your help.

Happy Friday!
Trent

Trent Gillaspie

Rackspace | Product Manager | 303.386.6115

Strategic | Futuristic | Communication | Responsibility | Arranger

### PR DESCRIPTION
**Pull request by stephamon**
Add Shared IP FAQs to Networks FAQ page

Fixes #1328

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e2371a4a0e/how-to/)

File: content/cloud-networks/cloud-networks-faq.md [hunk 1]
```diff
@@ -259,11 +259,72 @@ Vyatta appliance.
...
+
+[Cloud Networks API Guide](https://developer.rackspace.com/docs/cloud-networks/v2/developer-guide/)
+
+The Shared IP sections are:
+
+- [Concepts](https://developer.rackspace.com/docs/cloud-networks/v2/developer-guide/#shared-ip-addresses)
+
+- [API operations](https://developer.rackspace.com/docs/cloud-networks/v2/developer-guide/#document-api-operations/shared-ip-address-operations)
+
+- [Getting started](https://developer.rackspace.com/docs/cloud-networks/v2/developer-guide/#using-floating-ip-addresses)
```
Comment (catlook) [inline content/cloud-networks/cloud-networks-faq.md]: link should be: https://developer.rackspace.com/docs/cloud-networks/v2/developer-guide/#sharing-ip-address-between-servers

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a59c17c50d/how-to/). :bow:

**Comment by catlook** [comment]
👍

**Pull request merged**